api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -33,4 +33,18 @@
     	<renameColumn tableName="allergy" oldColumnName="comment" newColumnName="comments" columnDataType="varchar(1024)"/>
     </changeSet>
 
+    <changeSet id="TRUNK-5413" author="odorajonathan" dbms="mysql">
+      <preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="order_set" columnName="category"/></not>
+      </preConditions>
+      <comment>
+          Add Category field to OrderSet
```

Comment:
Can you change the comments to match the exact field name and database table name?